### ISSUE TITLE
 Beta 2021.12.0b0 Error

### ISSUE THREAD

**Issue by Wibias** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#196]
Entity camera.xiaomi_cloud_map_extractor (<class 'custom_components.xiaomi_cloud_map_extractor.camera.VacuumCamera'>) implements device_state_attributes. Please report it to the custom component author.

**Comment by Wibias** [comment]
Also this:
```
/config/custom_components/xiaomi_cloud_map_extractor/camera.py:148: DeprecationWarning: Call to deprecated function __init__ (This class will become the base class for all vacuum implementations. Use RoborockVacuum to control roborock vacuums.).
  self._vacuum = miio.Vacuum(host, token)
```

**Comment by hmmbob** [comment]
Requested more info in the #beta channel on Discord, got this info:

![image](https://user-images.githubusercontent.com/33529490/144705006-940aea05-2c58-4a53-aca6-6fcaec2f0ed0.png)

**Comment by PiotrMachowski** [comment]
Thanks @hmmbob! It looks like it was not included in breaking changes

**Comment by Wibias** [comment]
So yeah.. Your custom_components its causing some DeprecationWarnings in HA-Core component Harmony and pychromecast. Read here. https://github.com/home-assistant/core/issues/60952#issuecomment-988803722 Would help myself but I am not really sure on what to search for.

**Comment by PiotrMachowski** [comment]
@Wibias damn, this is weird... I need to investigate it

**Comment by hmmbob** [comment]
I can confirm - disabling this component stops throwing the depr warnings (which are sent as debug AFAIK). See also https://github.com/home-assistant/core/issues/61336

@PiotrMachowski probably this line?

https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/blob/1abdb95d7be5657729d089dadd3956cc0db232c2/scripts/get_map.py#L80

**Comment by PiotrMachowski** [comment]
@hmmbob this file is not included in the integration

**Comment by hmmbob** [comment]
I feel stupid 😬

**Comment by PiotrMachowski** [comment]
I will investigate it, just give me some time ;)

**Comment by joe248** [comment]
I'm seeing this in my log periodically after upgrading to 2021.12:

`DeprecationWarning:
    with timeout() is deprecated, use async with timeout() instead`

It doesn't specify which integration the error pertains to, but when I uninstall the cloud map extractor it goes away.

**Comment by PiotrMachowski** [comment]
Problem should be solved in v2.1.2

Not sure about other integrations though (https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/issues/196#issuecomment-988805353)
